aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -566,11 +567,11 @@ protected void removeOldTags(
             final ProgressEvent<ResourceModel, CallbackContext> progress
     ) {
         return proxy.initiate(
-                        "rds::reboot-db-instance",
-                        rdsProxyClient,
-                        progress.getResourceModel(),
-                        progress.getCallbackContext()
-                ).translateToServiceRequest(Translator::rebootDbInstanceRequest)
+                "rds::reboot-db-instance",
+                rdsProxyClient,
+                progress.getResourceModel(),
+                progress.getCallbackContext()
+        ).translateToServiceRequest(Translator::rebootDbInstanceRequest)
```

Comment:
I think the old indentation was correct

Reply:
Fixed. I think it is auto formatter in my IntelliJ. We should align our formatting rules.